content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,819 @@
+`github.com/aws/aws-sdk-go-v2/service/<service name>` | Service Client Module
+`github.com/aws/aws-sdk-go-v2/feature/s3/manager` | {{% alias service=S3 %}} Transfer Manager
+
+## Configuration Loading
+
+The [session]({{< apiref v1="aws/session" >}}) package and associated functionality are removed in the
+{{% alias sdk-go %}}, and are replaced with a simplified configuration system provided by the
+[config]({{< apiref "config" >}}) package. The `config` package is a separate Go module, and can be included in your
+applications dependencies by performing a `go get github.com/aws/aws-sdk-go-v2/config`.
+
+[session.New]({{< apiref v1="aws/session#New" >}}), [session.NewSession]({{< apiref v1="aws/session#NewSession" >}}),
+[NewSessionWithOptions]({{< apiref v1="aws/session#NewSessionWithOptions" >}}), and
+[session.Must]({{< apiref v1="aws/session#Must" >}}) must be migrated to
+[config.LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}).
+
+The `config` package provides several helper functions that aid in overriding the shared configuration loading
+programmatically. These function names are prefixed with `With` followed by option that they override. Let's look at
+some examples of how to migrate usage of the `session` package. For more information on loading shared configuration
+see [Configuring the {{% alias sdk-go %}}]({{% ref "configuring-sdk" %}}).
+
+#### Examples
+#####  Migrating from NewSession
+In this example usage of `session.NewSession` without additional argument parameters is migrated to
+`config.LoadDefaultConfig`.
+
+###### V1
+
+```go
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession()
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	// handle error
+}
+```
+
+##### Migrating from NewSession with aws.Config options.
+In this example we will show how to migrate overriding of `aws.Config` values during configuration loading. One or more
+`config.With*` helper functions can be provided to `config.LoadDefaultConfig` to override the loaded configuration
+values. In this example the AWS Region is overridden to `us-west-2` using the
+[config.WithRegion]({{< apiref "config#WithRegion" >}}) helper function.
+
+###### V1
+```go
+import "github.com/aws/aws-sdk-go/aws"
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession(aws.Config{
+	Region: aws.String("us-west-2")
+})
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(config.WithRegion("us-west-2"))
+if err != nil {
+	// handle error
+}
+```
+
+#####  Migrating from NewSessionWithOptions
+In this example we will show how to migrate overriding values during configuration loading. One or more
```

Comment:
Agian, I'd simplify to "This example shows..." or similar. Not crucial, but cleaner.